.github/workflows/run-a-single-node-from-branch.yml

```diff
@@ -0,0 +1,156 @@
+name: 'Run a node with selected configuration. NodeName=\"DevNode-${{ github.actor }}-${{ github.ref }}\"'
+
+on:
+  workflow_dispatch:
+    inputs:
+      network:
+        description: "Select a network on which You want to run a node"
+        default: "mainnet"
+        required: true
+        type: choice
+        options:
+          - mainnet
+          - gnosis
+          - sepolia
+          - goerli
+          - chiado
+      cl_client:
+        description: "Select Consensus Layer Client to run node against"
+        default: ""
+        required: true
+        type: choice
+        options:
+          - lighthouse
+          - lodestar
+          #- nimbus
+          - prysm
+          - teku
+      cl_custom_image:
+        description: "In case of need to run non-default cl image (different than actually supported by Sedge) put it in there"
+        default: ""
+        required: false        
+      config:
+        description: "Select a config file which will be selected for tests."
+        default: "default.json"
+        required: true        
+        type: choice
+        options:
+          - default.json
+          - fastSync.json
+          - fullSync.json
+          - fuzzer.json
+      timeout:
+        description: "Timeout in hours before triggering the deletion of smoke test instances. Maximum time of node lifetime can be 72 hours."
+        default: "24"
+        required: true
+      additional_nethermind_flags:
+        description: "Provide any additional flags to the Nethermind in space-separated format. Example: \"JsonRpc.Enabled=false Sync.SnapSync=false\"."
+        default: ""
+        required: false
+      additional_cl_flags:
+        description: "Provide any additional flags to the CL client in space-separated format. Example: \"clflag1=1 clflag2=2\"."
+        default: ""
+        required: false
+      
+jobs:
+  create_docker_image:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Trigger Docker Build Action
+      env:
+        REPOSITORY_DISPATCH_TOKEN: ${{ secrets.REPOSITORY_DISPATCH_TOKEN }}
+      run: |
+        curl -X POST \
+          -H "Accept: application/vnd.github+json" \
+          -H "Authorization: Bearer $REPOSITORY_DISPATCH_TOKEN" \
+          "https://api.github.com/repos/NethermindEth/nethermind/actions/workflows/build-nethermind-docker-images.yml/dispatches" \
+          -d '{
+            "ref": "${{ github.ref }}",
+            "inputs": {
+              "repo": "nethermindeth/nethermind",
+              "tag": "${{ github.ref }}",
+              "dockerfile": "Dockerfile"
+            }
+          }'
+
+    - name: Wait for Docker Build Action to complete
+      env:
+        GITHUB_TOKEN: ${{ secrets.REPOSITORY_DISPATCH_TOKEN }}
+      run: |
+        # Replace `workflow_id` with the ID of the workflow you want to wait for.
+        workflow_id="8166053" #Id of Docker build workflow
```

Comment:
@FalcoXYZ Yes - this will only trigger one specific action and URL asks for workflow_id. I think action Id should not change - if it is then should eb adjusted here as well. But script will fail and will not create anything then so should be safe and easy to catch.

Reply:
Ahh sorry!
@FalcoXYZ this is the one:
https://github.com/NethermindEth/nethermind/actions/workflows/build-nethermind-docker-images.yml